app/views/layouts/career_site/_footer.html.slim

```diff
@@ -1,4 +1,13 @@
-footer.pt-2.px-4
+footer.pt-2.pb-3.px-4
   .container-fluid.container-md.container-md-small
-    .text-center.text-md-start
-      = t("career_site.footer_text")
+    .row.row-gap-3
+      .col-12.col-md-6.text-center.text-md-start
+        = link_to t("career_site.footer_text"), "https://www.freeats.com/", target: "_blank"
+      .col-12.col-md-6.d-flex.justify-content-center.justify-content-md-end
+        - if ApplicationController::TOS_LINK.present?
+          = link_to t("career_site.tos_link"), ApplicationController::TOS_LINK, target: "_blank"
+        - if ApplicationController::TOS_LINK.present? && \
+             ApplicationController::PRIVACY_LINK.present?
+          .mx-2 ·
```

Comment:
There is an example on the tabler site of how the dot markup should look like
![image](https://github.com/user-attachments/assets/985da9cb-64bd-4a29-a631-da6d40145381)